lib/compress/zstd_compress_internal.h

```diff
@@ -1562,6 +1656,11 @@ void ZSTD_resetSeqStore(SeqStore_t* ssPtr);
  *  as the name implies */
 ZSTD_compressionParameters ZSTD_getCParamsFromCDict(const ZSTD_CDict* cdict);
 
+/*! ZSTD_checkCCtxCParams_internal() :
+ *  Checks the CParams in the CCtxParams (including related parameters not
+ * *actually* stored in the CParams struct). */
```

Comment:
maybe name them ?
It seems to be limited to `params->windowFrac` at this point.